packages/ffmpeg.rb

```diff
@@ -56,6 +56,7 @@ class Ffmpeg < Package
   depends_on 'zeromq'
   depends_on 'dav1d'
   depends_on 'libva'
+  depends_on 'intel_media_driver' if `grep -c 'GenuineIntel' /proc/cpuinfo`.to_i > 0
```

Comment:
Would this require new binaries for i686 and x86_64?  If so, ffmpeg is probably due for an update anyway.

Reply:
@supechicken: Thank you for attempting the upgrade.  ffmpeg has a bunch of dependencies and I imagine they need upgrades too.  Maybe try building a foundation of updated underlying dependencies first?